src/styles/blocks/card.scss

```
@@ -0,0 +1,87 @@
+.card {
+  width: $card-width;
+  background: #fff;
```

don't forget to create variables for all colors
```suggestion
  background: $c-white;
```